ConvivaIntegration/components/conviva/ConvivaAnalyticsTask.brs

```diff
@@ -34,7 +34,7 @@ sub internalInit()
   end if
 
   m.adTrackingMode = mapAdTrackingMode(m.top.config.adTrackingMode)
```

Comment:
m.top.config.adTrackingMode can be invalid 🤔 We should set BASIC as default value in this case